apt_sysvinit-utils.dockerfile

```diff
@@ -5,12 +5,19 @@ FROM $OS_TYPE:$BASE_IMAGE_TAG
 
 ENV container docker
 ENV DEBIAN_FRONTEND noninteractive
+ENV OS_TYPE debian
+ENV BASE_IMAGE_TAG 12
```

Comment:
@skyamgarp 👋 don't hardcore ENV values here as this is a dockerfile which is set programmatically for a number of `apt` based distros.
You can take an approach similiar to that [here](https://github.com/puppetlabs/litmusimage/blob/825d82ecedf00b14fcecf7fc512e33dcbf673276/yum_systemd.dockerfile#L1C1-L8C19) in how the ARGS are redefined after the `FROM` to make them once again available.
You'll then be able to use them like this [conditional](https://github.com/puppetlabs/litmusimage/blob/825d82ecedf00b14fcecf7fc512e33dcbf673276/yum_systemd.dockerfile#L14C1-L19C3).

Reply:
Yes, I have tried same, I just hard coded for testing.